internal/pkg/service/buffer/api/service/service_test.go

```diff
@@ -48,9 +49,11 @@ func TestParseRequestBody_Form(t *testing.T) {
 func TestParseRequestBody_TooLarge(t *testing.T) {
 	t.Parallel()
 
-	r := io.NopCloser(strings.NewReader(idgenerator.Random(1000009)))
+	size := int(datasize.MB + 1)
+	assert.Equal(t, 1024*1024+1, size)
+	r := io.NopCloser(strings.NewReader(idgenerator.Random(size)))
 	_, err := parseRequestBody("application/x-www-form-urlencoded", r)
-	assert.EqualError(t, err, "Payload too large.")
+	assert.EqualError(t, err, "Payload too large, the maximum size is 1MB.")
```

Comment:
Opraveny test.

Matematicky/vedecky su jednotky definovane takto:
```
1MB  = 10^6 = 1 000 000
1MiB = 2^20 = 1 048 576
```

Realne sa ale `1MiB` a `1MB` zamienaju.
Aj `datasize.MB = 1 048 576`.

Preto som ako limit zvolil `datasize.MB = 1 048 576`, teda vacsiu z moznosti.
Cisto prakticky, aby nam niekto nepisal, ze mu nejde nahrat `999kB (kiB)` a neprejde mu to.